sdk/cosmosdb/cosmos/src/client/Container/ContainerRequest.ts

```diff
@@ -1,8 +1,10 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import { ContainerDefinition } from "./ContainerDefinition";
+import { PartitionKeyDefinition } from '../../documents';
 
-export interface ContainerRequest extends ContainerDefinition {
+export interface ContainerRequest extends Omit<ContainerDefinition, 'partitionKey'> {
```

Comment:
We have some tests that ensure we are backwards compatible to TS 3.1. This is why `Omit` doesn't work.